scripts/variantstore/wdl/extract/curate_input_array_files.py

```diff
@@ -0,0 +1,48 @@
+import numpy as np
+from contextlib import contextmanager
+
+SAMPLE_MAP_FILE_SUFFIX = "sample_map_file"
+SAMPLE_NAME_FILE_SUFFIX = "sample_name_list_file"
+VCF_FILE_SUFFIX = "vcf_list_file"
+VCF_INDEX_FILE_SUFFIX = "vcf_index_list_file"
+
+@contextmanager
+def handle_file_error(file_name):
+    try:
+        yield
+    except:
+        print(f"ERROR: required file named '{file_name}' does not exist.")
```

Comment:
does this just soldier on despite the file not existing?

Reply:
It will check each file, output the message for each file that's missing and then exit.